doc/examples.md

```diff
@@ -106,7 +106,7 @@ This example Ignition configuration will locate the device with the "VAR" filesy
 }
 ```
 
-The SHA512 sum of the file can be determined using `sha512sum`.
+The SHA512 sum of the file can be determined using `sha512sum`. It also supports the SHA256 format which is determined using `sha256sum`.
```

Comment:
`It also supports` has an unclear referent.  What about "SHA256 sums are also supported, and can be calculated using `sha256sum`."?